components/scream/src/physics/p3/atmosphere_microphysics.hpp

```diff
@@ -84,6 +84,11 @@ class P3Microphysics : public AtmosphereProcess
 
   ekat::ParameterList     m_p3_params;
 
+  // Keep track of field dimensions and the iteration count
```

Comment:
Do we explain what sort of "iteration" is going on anywhere? If not, can we add a little comment that does?

Reply:
I removed m_it since it can now be stored in `infrastructure`, see line 190.  I also added a comment describing what it is even used for.  Question, would it make more sense to add that comment where the `P3Infrastructure` structure is defined?

Reply:
Yes--if the iteration count is stored in a field for that struct, the documentation for it belongs there.